src/status_im2/contexts/share/view.cljs

```diff
@@ -0,0 +1,163 @@
+(ns status-im2.contexts.share.view
+  (:require [utils.i18n :as i18n]
+            [quo2.core :as quo]
+            [react-native.core :as rn]
+            [react-native.safe-area :as safe-area]
+            [status-im2.contexts.share.style :as style]
+            [utils.re-frame :as rf]
+            [reagent.core :as reagent]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.qr-code.qr :as qr]
+            [clojure.string :as string]
+            ;;TODO(siddarthkay) : move the components below over to status-im2 ns
+            ;; issue -> https://github.com/status-im/status-mobile/issues/15549
+            [status-im.ui.components.list-selection :as list-selection]
+            [status-im.ui.components.react :as react]))
+
+(def ^:const profile-tab-id 0)
+(def ^:const wallet-tab-id 1)
+
+(defn copy-text-and-show-toast
+  [{:keys [text-to-copy post-copy-message]}]
+  (react/copy-to-clipboard text-to-copy)
+  (rf/dispatch [:share/show-successfully-copied-toast post-copy-message]))
+
+(defn header
+  []
+  [rn/view
+   [quo/button
+    {:icon                true
+     :type                :blur-bg
+     :size                32
+     :accessibility-label :close-activity-center
+     :override-theme      :dark
+     :style               style/header-button
+     :on-press            #(rf/dispatch [:hide-popover])}
+    :i/close]
+   [quo/text
+    {:size   :heading-1
+     :weight :semi-bold
+     :style  style/header-heading}
+    (i18n/label :t/share)]])
+
+(defn profile-tab
+  [window-width]
+  (let [multiaccount              (rf/sub [:multiaccount])
+        emoji-hash                (string/join "" (get multiaccount :emoji-hash))
+        qr-size                   (- window-width 64)
+        public-pk                 (get multiaccount :public-key)
+        profile-qr-url            (str "https://join.status.im/u/" public-pk)
```

Comment:
Consider using a constant @siddarthkay. I think I've seen this hostname in a bunch of places, ideally we should have a proper place for building important URLs in the system.

Reply:
resolved here: https://github.com/status-im/status-mobile/pull/15528/commits/6dd9897540ca2ab37b0d190dfae3cd80ddb1b56d

Reply:
> we should have a proper place for building important URLs

I was thinking for media-server and/or QR code related components, we could take care of building use-case specific URLS  in the common component itself.